pgxn/neon/walproposer.c

```diff
@@ -425,7 +425,7 @@ WalProposerPoll(void)
 
 			break;
 		}
-		
+
```

Comment:
These changes seem unrelated to me. can you remove them from this PR? Maybe put them into a separate one?